schema/migration-2-0035-20240108.sql

```diff
@@ -0,0 +1,29 @@
+-- Persistent generated migration.
+
+CREATE FUNCTION migrate() RETURNS void AS $$
+DECLARE
+  next_version int ;
+BEGIN
+  SELECT stage_two + 1 INTO next_version FROM schema_version ;
+  IF next_version = 35 THEN
+    EXECUTE 'alter domain addr29type drop constraint addr29type_check' ;
```

Comment:
When we create these domains
```
CREATE DOMAIN addr29type AS bytea CHECK (octet_length (VALUE) = 29);
```
we don't give any name to the constraint. `addr29type_check` is the name that postgres gives by default.